### PR TITLE
Describe suspend and resume behavior.

### PR DESCRIPTION
**Pull request by mfoltzgoogle**
This addresses #107 and #108. 

For #107, I think the language about keeping cached mDNS records should be good enough.

For #108, we probably need a way for different capabilities in the protocol to "re-sync" their state after a network disconnection.  For example, a remote playback receiver would need to send a `remote-playback-state-event` message to update the sender on what the playback is doing. I'll have to think more about how to handle that scenario across the different capabilities in the protocol.

***
<a href="https://pr-preview.s3.amazonaws.com/w3c/openscreenprotocol/pull/318.html" title="Last updated on Oct 16, 2023, 4:38 PM UTC (a173b61)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/openscreenprotocol/318/dcc0efb...a173b61.html" title="Last updated on Oct 16, 2023, 4:38 PM UTC (a173b61)">Diff</a>

### CODE REVIEW

File: index.bs [hunk 1]
```diff
@@ -322,6 +324,7 @@ it should change the DNS-SD service name to a new value, indicating a new
 mechanism for metadata discovery.
 
 
+
```
Comment (baylesj) [inline index.bs:327]: Needed? Three empty lines seems like a lot.

**Review by baylesj — APPROVED** [review]
LGTM % feedback.

File: index.bs (hunk 1 above)
Comment (mfoltzgoogle) [reply index.bs:327]: Removed one line.

**Comment by mfoltzgoogle** [comment]
Thanks @baylesj, I merged based on your LGTM.

**Pull request merged**